deploy/addons/kubevirt/pod.yaml.tmpl

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: kube-system
+  name: kubevirt-scripts
+  labels:
+    kubernetes.io/minikube-addons: kubevirt
+    addonmanager.kubernetes.io/mode: Reconcile
+data:
+  uninstall.sh: |
+    #!/bin/bash
+
+    kubectl delete -f /manifests/kubevirt.yaml
+
+    kubectl delete -f /manifests/kubevirt-base.yaml
+
+  install.sh: |
+    #!/bin/bash
+
+    export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases | grep tag_name | grep -v -- - | sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs)
```

Comment:
Ha. Nifty, nice to see that this is _not_ the naive approach of retrieving the highest version :)